source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/index.tsx

```diff
@@ -11,36 +11,24 @@ export interface IProps {
   className?: string;
 }
 
-interface IState {
-  isOrgSwitcherActive: boolean;
+export default function Sidebar({ closeSidebar, className }: IProps) {
```

Comment:
converted this to a functional component to (try to ) get rid of a state mutation after unmount